bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/EntryLogger.java

```diff
@@ -72,7 +80,7 @@
  * the actual ledger entry. The entry log files created by this class are
  * identified by a long.
  */
-public class EntryLogger {
+public class EntryLogger implements EntryLoggerIface {
```

Comment:
Please rename this class to DefaultEntryLogger
and name the interface 'EntryLogger' because EntryLoggerIFace does not sound good in Java